Variances

MalamaKangwa opened the pull request.

MalamaKangwa committed:
Feature: Variance Added

MalamaKangwa committed:
Update: Updated with Development

Pacanor commented on StatisticOperations/StatVariance.cs:
```diff
@@ -0,0 +1,33 @@
+        {
+
+            dynamic theMean = StatMean.Mean(arrayA);
+            int arrayCount_1 = arrayA.Length - 1;
```
arrayA.Length - 1 is for samples. We are doing calculations for populations. Please change this before merging

Pacanor committed:
Fix: Changed to Population Variance

Removed the - 1 which changes it from a sample variance to a population variance

Pacanor reviewed (approved):
Review the change that I've made and merge it with your branch. Other than that, the code is fine

MalamaKangwa committed:
Merge pull request #28 from Pacanor/Pacanor-variance-patch-1

Fix: Changed to Population Variance
